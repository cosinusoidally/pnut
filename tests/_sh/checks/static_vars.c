// expect_comp_failure

static int some_var; // Allowed

void main() {
  static int some_other_var; // Static local vars are not supported
}
