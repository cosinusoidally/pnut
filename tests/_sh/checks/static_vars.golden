tests/_sh/checks/static_vars.c:8:0  Extern and static storage class specifier not supported on local variables
